src/DependencyInjection/Factory/CoreAdminFactory.php

```diff
@@ -34,26 +34,21 @@ public function getKey()
     public function addConfiguration(NodeBuilder $builder)
     {
         $builder
-            ->scalarNode('form_group')->defaultValue('form.group_core')->end()
             ->arrayNode('extensions')
                 ->addDefaultsIfNotSet()
                 ->children()
                     ->arrayNode('publishable')
                         ->addDefaultsIfNotSet()
                         ->children()
                             ->scalarNode('form_group')->defaultValue('form.group_publish_workflow')->end()
+                            ->scalarNode('form_tab')->defaultValue('form.tab_publish')->end()
                         ->end()
                     ->end()
                     ->arrayNode('publish_time')
                         ->addDefaultsIfNotSet()
                         ->children()
                             ->scalarNode('form_group')->defaultValue('form.group_publish_workflow')->end()
-                        ->end()
-                    ->end()
-                    ->arrayNode('translatable')
```

Comment:
what about translatable? or is this now completely handled by SonataTranslationBundle?